lib/cloud_controller/config_schemas/api_schema.rb

```diff
@@ -274,6 +274,9 @@ class ApiSchema < VCAP::Config
               ca_file: String,
               cert_file: String,
               key_file: String,
+              optional(:connect_timeout) => Integer,
```

Comment:
Is there a corresponding [capi-release](https://github.com/cloudfoundry/capi-release) PR to allow for these to be configurable from the BOSH job? I glanced briefly, but wasn't able to find one.

Reply:
Ah, my mistake yes. Forgot to actually open that. Here it is: https://github.com/cloudfoundry/capi-release/pull/110